devtools/cmd/migrate-sidekick/main_test.go

```diff
@@ -414,3 +417,47 @@ func TestBuildConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestRunMigrateCommand(t *testing.T) {
+	t.Parallel()
+	for _, test := range []struct {
+		name    string
+		path    string
+		wantErr string
+	}{
+		{
+			name: "success",
+			path: "testdata/root-sidekick/success",
+		},
+		{
+			name:    "tidy_command_fails",
+			path:    "testdata/tidy-fails",
+			wantErr: "librarian tidy command failed",
```

Comment:
if you want to check for a specific error, I would use a named error (https://pkg.go.dev/errors#New), and check with https://pkg.go.dev/errors#Is, so that if the string changes tests don't break. Checking for error strings can be brittle